pipeline_dp/pipeline_backend.py

```diff
@@ -132,6 +132,10 @@ def flatten(self, col1, col2, stage_name: str):
         """
         pass
 
+    def annotate(self, col, **kwargs):
```

Comment:
Beam needs `stage_name` for each `PTransform`, please add `stage_name:str` (the same as for others methods of `PipelineBackend`)